editor/plugins/packed_scene_translation_parser_plugin.cpp

```diff
@@ -52,32 +52,53 @@ Error PackedSceneEditorTranslationParserPlugin::parse_file(const String &p_path,
 	Ref<SceneState> state = Ref<PackedScene>(loaded_res)->get_state();
 
 	Vector<String> parsed_strings;
+	List<Pair<NodePath, bool>> atr_owners;
 	List<String> tabcontainer_paths;
 	for (int i = 0; i < state->get_node_count(); i++) {
 		String node_type = state->get_node_type(i);
-		bool is_control = ClassDB::is_parent_class(node_type, "Control");
-		if (!is_control && !ClassDB::is_parent_class(node_type, "Window")) {
-			continue;
-		}
+		String parent_path = state->get_node_path(i, true);
 
-		// Find the `auto_translate` property, and abort the string parsing of the node if disabled.
+		// Find the `auto_translate_mode` property, and abort the string parsing of the node if disabled, or inheriting a disabled one.
 		bool auto_translating = true;
 		for (int j = 0; j < state->get_node_property_count(i); j++) {
-			if (state->get_node_property_name(i, j) == "auto_translate" && (bool)state->get_node_property_value(i, j) == false) {
-				auto_translating = false;
-				break;
+			if (state->get_node_property_name(i, j) != "auto_translate_mode") {
+				continue;
 			}
+
+			int idx_last = atr_owners.size() - 1;
+			if (idx_last > 0 && !parent_path.begins_with(atr_owners[idx_last].first)) {
```

Comment:
Ooof, accessing list elements using index. I think Vector or LocalVector would be better suited here.